src/opnsense/www/js/widgets/Picture.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2024 Deciso B.V.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in the
+ *    documentation and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES,
+ * INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY
+ * AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ * AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY,
+ * OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+export default class Picture extends BaseWidget {
+    constructor(config) {
+        super(config);
+    }
+
+    getMarkup() {
+        return $('<div id="picture-container" style="padding: 5px;"><img id="picture" style="width: 100%; height: 100%;"></div>');
+    }
+
+    async onMarkupRendered() {
+        $('#picture').attr('src', '/system_general.php?getpic=true');
```

Comment:
this won't lint with ajaxCall requirement :D

Reply:
I can live with the exception to the rule 😶

Reply:
it would make sense to add an endpoint or use an existing one: the user can only fetch its own picture anyway

Reply:
it's inherent to dashboard settings really...

Reply:
still, moving this to ajaxCall requires modifiying it as it only does `json` at the moment

Reply:
/ui/... is also a possibility although I guess it does't really matter. I can add parsing this to the linter too

Reply:
@fichtner How about https://github.com/opnsense/core/pull/7860/commits/e376affedcc90c82a6aaee1b244cc71470be60b1?